tools/cxbe/Main.cpp

```diff
@@ -54,6 +69,87 @@ int main(int argc, char *argv[])
         szXbeTitle[40] = '\0';
     }
 
+    // interpret title id
+    if(szXbeTitleID[0])
+    {
+        bool hex = true;
+        for(int i = 0; szXbeTitleID[i]; ++i)
+        {
+            if(szXbeTitleID[i] == '-')
+            {
+                hex = false;
+                break;
+            }
+        }
+        if(hex)
+        {
+            sscanf(szXbeTitleID, "%x", &dwTitleId);
+        }
+        else
+        {
+            char titlechar[2] = { 'C', 'X' };
+            unsigned titleno = 9999;
+            sscanf(szXbeTitleID, "%c%c-%u", &titlechar[0], &titlechar[1], &titleno);
+            if(titleno > 0xFFFF)
+            {
+                printf("WARNING: Title ID number too high (max is 65535)\n");
+                titleno = 0xFFFF;
+            }
+            dwTitleId = (titlechar[0] << 24) | (titlechar[1] << 16) | titleno;
+        }
+    }
+
+    // interpret region flags
+    if(szXbeRegions[0])
+    {
+        char c;
+        for(int i = 0; (c = szXbeRegions[i]); ++i)
+        {
+            switch(c)
+            {
+                case '-':;
+                    dwRegions = 0;
```

Comment:
This shouldn't have to set to zero?
I'd expect that the user has to add regions like `-REGION:nj` or similar. I'd expect default to be `njwm`.

Reply:
`-` means no regions and an empty string means all

Reply:
Wouldn't `-REGION: -TITLE:Foo` be empty? or is that an issue with the CLI arg parser?